crates/esplora/tests/async_ext.rs

```diff
@@ -109,6 +109,19 @@ pub async fn test_update_tx_graph_without_keychain() -> anyhow::Result<()> {
         )
         .await?;
 
+    // Check to see if we have the floating txouts available from our two created transactions'
+    // previous outputs in order to calculate transaction fees.
+    let fee_calculations = graph_update
+        .full_txs()
+        .map(|tx| graph_update.calculate_fee(tx.tx).is_ok())
```

Comment:
I think we should check whether the fee calculation is correct (instead of whether a fee calculation has occurred).

You can either use Esplora's [`GET /tx/:txid`](https://github.com/Blockstream/esplora/blob/master/API.md#get-txtxid) endpoint or bitcoind RPC's `gettransaction` call to get the fee.